### DISCUSSION TITLE
Debugging rejected breakpoints

### DISCUSSION THREAD

**Discussion by jeanlucthumm** [mfussenegger/nvim-dap#276] [Q&A] [answered]
I'm trying to setup `lldb-vscode` to debug the rust compiler as its compiling something. So far, I've gotten `dap` to launch the compiler on `continue()` and it actually goes through and compiles the target package. However, it doesn't seem to be stopping on any breakpoints in the compiler source code. After the run, the breakpoints turns into an `R`:

![image](https://user-images.githubusercontent.com/4934853/129640035-ffea32a4-7ebd-49dd-ac52-89840c7a510a.png)

I read this means it was rejected. Does `nvim-dap` keep any logs for why it was rejected?

**Comment by mfussenegger** [comment, marked as answer]
I suspect you might be missing debug symbols.

See `:help dap.set_log_level()` for information regarding logging.
